flocker/node/_config.py

```diff
@@ -1289,14 +1307,14 @@ def current_from_configuration(current_configuration):
     return Deployment(nodes=frozenset(nodes))
 
 
-def marshal_configuration(state):
+def marshal_node_state(state):
     """
-    Generate representation of a node's applications using only simple Python
-    types.
+    Generate representation of a node's current node state using
+    only simple Python types.
 
-    :param NodeState state: The configuration state to marshal.
+    :param NodeState state: The deployment state to marshal.
```

Comment:
Considering the explicit and sole purpose of a "model" type is to represent state, I find the name `NodeState` somewhat redundant.  Guess that point isn't really in scope here though.